### DISCUSSION TITLE
Error installing package

### DISCUSSION THREAD

**Discussion by DanniAnderson** [IBAHCM/RPiR#294] [Archive] [answered]
Hi,

I was trying to install the someone's package but I get this error...

`devtools::install_github("annabellockeBCI")`

> Error in parse_repo_spec(repo) :
> Invalid git repo specification: 'annabellockeBCI'

I'm not sure how to sort this.

Thanks

**Comment by soniamitchell** [comment, marked as answer]
Hi @DavidAnderson102, try `devtools::install_github("IBAHCM/annabellockeBCI")`

**Reply by DanniAnderson** [reply]
Thank you, that worked
